packages/app/components/Queue/Student/HotnessBar.tsx

```diff
@@ -0,0 +1,49 @@
+import { ReactElement } from 'react'
+import styled from 'styled-components'
+
+// couldn't figure out how to make the height of the bar dynamic
+const BAR_WIDTH_PX = 10
+const BAR_HEIGHT_PX = 50
+
+/**
+ * Renders a vertical bar that shows the hotness of a question.
+ * @param hotness a number between 0 (least hot) to 100 (most hot)
+ * @param hasGreen whether or not to have the variation where the bar goes from green to red (as one colour) vs white to red (as a gradient)
```

Comment:
Would it better to name it `hasVariation`?

Reply:
I thought about that and hasVariation (or similar) has the problem of not easily knowing what variation is which. Will rename to isGreenVariation